codalab/codalab-worksheets
# Invalid bundle reference in a  table with schema

#1819 This seems to be the easiest fix, ideally, the backend should change the return information though

To test: copy paste

% schema schema1
% add uuid uuid '[0:8]'
% add name
% add summary
% add metadata
% add permission
% add group_permissions
% display table schema1
[run run-date- : date; echo hello]{0x7740626c82874067b75d030a415292ab}
[run run-pip3 : pip3 list]{0x5f7d47919a5a49679073e3928386c99e}
[run run-echo : echo date]{0xe58dd5eac1a249b285421bb208af462e}
[run run-date : date]{0xf767161392ad4d10aa02e82a5b7b244c}

to any worksheet, no longer crashes
